### ISSUE TITLE
Radio button selection not displayed in Windows high contrast mode

### ISSUE THREAD

**Issue by JMGordon** [carbon-design-system/carbon#3184]
In the IBM Cloud portal the support case form is using the radio button component from `carbon-components-react@6.115.3`. In Windows high contrast mode radio button selection is not displayed, as can be seen in the below screen shot:
![unified-support-windows-high-contrast](https://user-images.githubusercontent.com/7243419/60196622-2e435800-9803-11e9-89b8-fddc68bfc6f9.png)

 
This issue also shows up in the carbon components react storybook:
![radio-button-windows-high-contrast](https://user-images.githubusercontent.com/7243419/60196295-6ac28400-9802-11e9-8624-fe69e4771fe7.jpg)

**Comment by johnsteinbacher** [comment]
This issue is holding up the Console AVT exit.  Because of that it is high priority for us.

**Comment by dakahn** [comment]
Looking into why our radio button's aren't picking up the HCM styles in our reset. Thanks for reporting. 👍

**Comment by dakahn** [comment]
Our HCM detecting CSS only applies to our SVG icons -- [in this case our radio buttons are CSS made](https://github.com/carbon-design-system/carbon/issues/1600#issuecomment-507468460) and will need some further styling a la

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-ms-high-contrast
http://www.gwhitworth.com/blog/2017/04/how-to-use-ms-high-contrast/

**Comment by dakahn** [comment]
related #1600

**Comment by dakahn** [comment]
Working on this currently. Watch this space for a PR, folks.

**Comment by dakahn** [comment]
Have a fix up for Edge/IE. Firefox is tough though -- the `-ms-high-contrast: active` selector isn't compatible there and supporting HCM formally might involve a larger refactor re-examining creating the active state _fill in_ effect with CSS only.

**Comment by dakahn** [comment]
I'm lowering the severity since we have a workaround with Edge and Chrome 👍

**Comment by dakahn** [comment]
With the aforementioned PR #3442 we should have HCM support for MS browsers as well as Chrome (via their high-contrast extension). **We do not support High Contrast Mode on Windows for Firefox due to that browsers lack of support for the necessary Microsoft CSS extension used to detect HCM.** Though there are Bugzilla proposals for a `prefers-contrast` media query [seen here](https://bugzilla.mozilla.org/show_bug.cgi?id=1506364). We will watch that issue and update accordingly 👍🏽

**Comment by elizabethsjudd** [comment]
@dakahn has this ☝️  been okay'd by the IBMa team (to post-pone support for Firefox)?  Is it possible to get that documented somewhere so our offerings are aware?

**Comment by dakahn** [comment]
@elizabethsjudd writing up something official we can point to as we speak 👍. 

It was a topic during the IBMa morning standup today and they reached consensus that Windows HCM support on Edge, IE, and Chrome was enough.

**Comment by dakahn** [comment]
wrote up some background at #3478
